README.md

```diff
@@ -43,90 +64,9 @@ Developers might want to install the code in place so that FASTSim files can be
     1. Run `pytest -v python/fastsim/tests/`
 
 # Usage
-To see and run examples, navigate to `./python/fastsim/demos` and run the various *demo.py files to see fastsim use cases. There are other examples in fastsim/tests.  
-
-
-# Adding FASTSim as a Depency in Rust
-## Via GitHub
-Add this line:  
-`fastsim-core = { git = "https://github.nrel.gov/MBAP/fastsim", branch = "rust-port" }`  
-to your Cargo.toml file, modifying the `branch` key as appropriate.  
-
-## Via Cargo
-This has not been implemented yet.  
-
-# List of Abbreviations
-cur = current time step  
-prev = previous time step  
-cyc = drive cycle  
-secs = seconds  
-mps = meters per second  
-mph = miles per hour  
-kw = kilowatts, unit of power  
-kwh = kilowatt-hour, unit of energy  
-kg = kilograms, unit of mass  
-max = maximum  
-min = minimum  
-avg = average  
-fs = fuel storage (eg. gasoline/diesel tank, pressurized hydrogen tank)  
-fc = fuel converter (eg. internal combustion engine, fuel cell)  
-mc = electric motor/generator and controller  
-ess = energy storage system (eg. high voltage traction battery)  
-chg = charging of a component  
-dis = discharging of a component  
-lim = limit of a component  
-regen = associated with regenerative braking  
-des = desired value  
-ach = achieved value  
-in = component input  
-out = component output  
-
-# New Features and Capabilities
-In `fastsim-3`, we introduce:
-- modular, hierchical component structure 
-- component-specific vehicle models -- i.e. the conventional vehicle contains only data for the fuel
-  converter and other relevant components but does contain any dummy battery or motor parameters as
-  is the case in `fastsim-2`
-- faster run times due to modularity and extensive streamlining
-- ability to control granularity of time-resolved data -- e.g. save at every time step, save at every _n_th time step, or never save at all (saving only cumulative trip-level results)
-- ability to simulate standalone component models
+To see and run examples, navigate to `./python/fastsim/demos` and run the various *demo.py files 
```

Comment:
@robinsteuteville , could you update this section to show how to copy the demo files locally using the function we made?  This will also need to happen in `fastsim-2` I think.

Reply:
@calbaker 
I will definitely add directions in the `fastsim-2` readme.
The function in `fastsim-2` that copied the demos was called `copy_demo_files`. I can't find a similar function in `fastsim-3`. Is it under a different name, or does it not exist yet? If it doesn't exist yet, should we wait until it does exist before writing instructions into the `fastsim-3` readme?